source/iterator/combined.go

```diff
@@ -81,6 +82,8 @@ type CombinedParams struct {
 	Position       *Position
 }
 
+const sourceOpenTimeRetries = time.Second * 8
```

Comment:
Let's delete this and the similar `destinationOpenTimeRetries` fields, and add a new optional field `ReconnectTimeout` to the `config.Config` struct along with a constant, `defaultReconnectTimeout`, which will be equal to `5s`.

And after you make these changes, please also add the new field to the tables in the readme.